runtime/oti/vmconstantpool.xml

```diff
@@ -239,6 +240,15 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<fieldref class="java/lang/reflect/Constructor" name="declaringClass" signature="Ljava/lang/Class;">
 		<fieldalias name="clazz"/>
 	</fieldref>
+	<fieldref class="java/lang/reflect/RecordComponent" name="declaringClass" signature="Ljava/lang/Class;">
+		<fieldalias name="clazz"/>
```

Comment:
why the fieldalias?  What does this give over using the existing name?

Reply:
following convention from  the declaringClass entries for Constructor/Field/Method classes.

Reply:
That may be a historical hold-over from when we had a separate implementation of Reflection.  I'd prefer not to add aliases unless they're required.